internal/coreinternal/metricstestutil/metric_diff.go

```diff
@@ -206,6 +206,7 @@ func diffHistogramPt(
 	diffs = diffMetricAttrs(diffs, expected.Attributes(), actual.Attributes())
 	diffs = diff(diffs, expected.Count(), actual.Count(), "HistogramDataPoint Count")
 	diffs = diff(diffs, expected.Sum(), actual.Sum(), "HistogramDataPoint Sum")
+	// TODO: HasSum, Min, HasMin, Max, HasMax are not covered in tests.
```

Comment:
Please open a separate issue to track this

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12951/files/5611572aed810144b005fd00f1fb668f93b17e5b#discussion_r994916101